Code/Engine/Core/System/Implementation/xcb/InputDevice_xcb.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <Core/Input/DeviceTypes/MouseKeyboard.h>
+
+class EZ_CORE_DLL ezStandardInputDevice : public ezInputDeviceMouseKeyboard
```

Comment:
Is this supposed to stay? I thought you wanted to use glfw instead?

Reply:
@Sanakan8472 wanted me to commit this intermediate state so he can cross check his future changes on linux. Once I have glfw fully working, I will remove the xcb implementation again.